extra/string-extra/Tests/Fixtures/singular-invalid-language.test

```diff
@@ -1,10 +1,10 @@
 --TEST--
 "singular" filter
 --TEMPLATE--
-{{ 'partitions'|singular('it') }}
+{{ 'partitions'|singular('qq') }}
 
 --DATA--
 return []
 
 --EXCEPTION--
-Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("Locale "it" is not supported.") in "index.twig" at line 2.
```

Comment:
Can you revert these changes were you've removed the end line?

Reply:
done